dev-haskell/haskell-language-server/haskell-language-server-0.8.0.0.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+# Tests fail
```

Comment:
In this case, it seems like the tests are failing due to missing files in the release tarball. Specifically, it seems like [the testdata](https://github.com/haskell/haskell-language-server/tree/master/test/wrapper/testdata) sub-directory is not included for some reason.

You can either update the comment to reflect this, or try to fetch the missing files yourself (again, I don't mind working on this once I'm done with the review)